src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenAsyncEHTests.cs

```diff
@@ -2104,5 +2104,43 @@ static void Main()
 ";
             CompileAndVerify(source, expected);
         }
+
+        [Fact, WorkItem(67091, "https://github.com/dotnet/roslyn/issues/67091")]
+        public void NestedCatch_DuplicateLocal()
```

Comment:
>NestedCatch_DuplicateLocal [](http://example.com/codeflow?start=20&length=26)

Is it possible to add a test with more levels of nesting? For example, where locals from at least two distinct parent ```AwaitCatchFrame```s would cause problems at the bottom ```AwaitCatchFrame``` before the change. Basically, if we think that it is required to pay attention to ```_hoistedLocals``` across the entire hierarchy, are we demonstrating this with a test?